common/app/layout/Breakpoint.scala

```diff
@@ -0,0 +1,19 @@
+package layout
+
+import scalaz.syntax.std.option._
+
+sealed trait Breakpoint {
+  val minWidth: Option[Int]
+}
+
+case object Mobile extends Breakpoint {
+  val minWidth = None
+}
+
+case object Tablet extends Breakpoint {
+  val minWidth = 740.some
```

Comment:
It's like ... one character shorter! I'm a very slow typer, so that's saved me at least a minute's worth of development time.

(The actual reason is that .some returns an Option[A] rather than Some[A]. This is sometimes important due to Scala's compiler being dumb, and I've gotten into the habit of doing it ... though I will refrain hereafter :walking:)